src/Runner/execTflite.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
how about using `execTFLite.py` or `exec_tflite.py` ?

Reply:
Every filename in this repo is following PascalCase.. I should change to `ExecTFLite.py` in my thought of naming rule. Any reason to use snake case?

Reply:
`ExecTFLite.py` looks good :)